src/pages/[locale]/examples/built-in-utilities/api.tsx

```diff
@@ -12,7 +12,7 @@ import withApollo from '../../../../hocs/withApollo';
 import { StaticParams } from '../../../../types/nextjs/StaticParams';
 import { OnlyBrowserPageProps } from '../../../../types/pageProps/OnlyBrowserPageProps';
 import { SSGPageProps } from '../../../../types/pageProps/SSGPageProps';
-import { getCommonStaticPaths, getCommonStaticProps } from '../../../../utils/nextjs/SSG';
+import { getExamplesCommonStaticPaths, getExamplesCommonStaticProps } from '../../../../utils/nextjs/SSG';
```

Comment:
What's the reason why you're renaming this?

Reply:
nvm, this explains it: https://github.com/UnlyEd/next-right-now/pull/68/commits/fd494c6896a377b42f6f9c176fc6e89059d2b900#diff-bd907cbc8270eada6403d4b6f15e05a6R21

Reply:
Yes, what do you think about that change BTW? I think it may help, even if it's not bullerproof.

Reply:
Yeah I think it does make sense, specially for newcomers.